slides/week2/week2.org

```diff
@@ -0,0 +1,90 @@
+#+TITLE: Week 2
+#+AUTHOR: Kristaps Berzinch, Raymond Ortiz
+#+EMAIL: kristaps@robojackets.org, rortiz9@gatech.edu
+
+* What are we doing today?
+- Introduction to C++
+ - What it is
+ - Functions
+ - Includes and headers
+ - Working with strings
+- How to find and read documentation
+- Sending and recieving changes from remote Git repositories
+
+* What is C++?
+- A general-purpose programming language
+- Designed with performance, efficiency, and flexibility in mind
+- Great for robots!
+
+* Functions
+#+BEGIN_NOTES
+After introducing the concept, write a simple example. Don't go into prototypes yet, we'll cover them in a bit.
+
+Open the example code on your display and try to stick to that as much as you can.
+#+END_NOTES
+- A function is a chunk of code with defined inputs and outputs
+ - Minimizes copy-and-paste
+ - Less buggy
+ - Easier to test than a whole program
+- These can be chained together for more complex functionality
+- You want each function to have a single high-level purpose
+
+* Side note: function overloading
+- C++ lets you have multiple functions with the same name, provided their parameter signatures are different
```

Comment:
I don't like types a lot (since the length matters too, not just types).

Maybe 'provided their argument lists are different'? I don't know if that's much better though.